beta-20161006.md

```diff
@@ -6,7 +6,7 @@ summary: Additions and changes in CockroachDB version beta-20161006.
 
 ## Oct 6, 2016
 
-Although this week's release contains only a few internal changes and doc updates, note that we are preparing to merge numerous features from our `develop` branch to `master` for inclusion in an upcoming release.  
+Although this week's release contains only a few internal changes and doc updates, note that we are preparing to release numerous features currently on our `develop` branch.
```

Comment:
not to be overly pedantic, but these are not on our develop branch; that branch is already deleted.

i would suggest avoiding being overly specific here.